lib/improver/tests/nowcasting/utilities/test_ApplyOrographicEnhancement.py

```diff
@@ -98,13 +98,6 @@ def test_basic(self):
         plugin = ApplyOrographicEnhancement("add")
         self.assertEqual(plugin.operation, "add")
 
-    def test_exception(self):
-        """Test that an exception is raised if the operation requested is
-        not a valid choice."""
-        msg = "Operation 'multiply' not supported for"
-        with self.assertRaisesRegex(ValueError, msg):
-            ApplyOrographicEnhancement("multiply")
-
```

Comment:
Well yes but can you think of any conceivable reason why a user might try to do this?  I mean, I can put it back, but it might as well say "peanuts".

Reply:
If I were writing the test, I would request an operation named "invalid" rather than "multiply". A user might hit this error by misspelling the word they wanted.

Reply:
Good point.  I'll change this.